docs/wetboek.md

```diff
@@ -803,5 +803,14 @@
 | **Tweede Veroordeling**  |   | 20 uur  | €6,000  |
 | **Meerdere Veroordelingen**  | 20 maanden  |  | €9,000  |
 
+### Artikel VIII-10 Lokaalvredebreuk (139 SR)
+
+1. Schuldig is een persoon die in een voor de openbare dienst bestemd lokaal, openbare ruimte of besloten erf dat bestemd is voor de openbare dienst, wederrechtelijk binnendringt, of, wederrechtelijk aldaar vertoevende, zich niet op de vordering van de bevoegde ambtenaar aanstonds verwijdert.
+2. Schuldig is een persoon die zich de toegang heeft verschaft door middel van braak of inklimming, van valse sleutels, van een valse order of een vals kostuum, of die zonder voorkennis van de bevoegde ambtenaar en anders dan ten gevolge van vergissing binnengekomen, aldaar wordt aangetroffen in de voor de nachtrust bestemde tijd, wordt geacht te zijn binnengedrongen.
+
+|   | Celstraf  | Taakstraf  | boete  |
+|  Eerste Veroordeling |   | 35 uur  | €1,500  |
+| Tweede Veroordeling  |   | 40 uur  | €2,000  |
```

Comment:
Deze missen in kosten pagina

Reply:
Is opgelost door een nieuwe pullrequest.